fullstop-jobs/src/main/java/org/zalando/stups/fullstop/jobs/ec2/FetchEC2Job.java

```diff
@@ -192,15 +193,15 @@ public void check() {
                             }
 
                             HttpGetRootCall httpCall = new HttpGetRootCall(httpclient, instancePublicIpAddress, allowedPort);
-                            ListenableFuture<Boolean> listenableFuture = threadPoolTaskExecutor.submitListenable(
+                            ListenableFuture<HttpCallResult> listenableFuture = threadPoolTaskExecutor.submitListenable(
                                     httpCall);
                             listenableFuture.addCallback(
-                                    result -> {
+                                    httpCallResult -> {
                                         log.info("address: {} and port: {}", instancePublicIpAddress, allowedPort);
-                                        if (!result) {
+                                        if (!httpCallResult.isSecured()) {
                                             Map<String, Object> md = newHashMap();
                                             md.put("instancePublicIpAddress", instancePublicIpAddress);
-                                            md.put("allowedPort", allowedPort);
```

Comment:
imho it is ok to leave the port, that has been called, in the meta data object. Just don't call it "allowedPort".